docs/submission_review.rst

```diff
@@ -0,0 +1,214 @@
+SubmissionReview
+****************
+
+This endpoint supports List, Retrieve, Update, Create Submission Reviews
+
+Where:
+
+- ``id`` - the id of the submission review (required when doing an update)
+- ``instance`` - the id of the Instance object being reviewed
+- ``note`` - the submission review comment
+- ``status`` - the submission review status. Should be one of:
+
+    - ``'1'`` - approved
```

Comment:
Does this mean it has to be a string? Can it be a number as well? Or should we better yet use the string `"approved" "rejected" "pending"` which are far much clearer?

Reply:
I would say will leave it as it is because the documentation explains what each digit means.